tests/basilisp/contrib/nrepl_server_test.lpy

```diff
@@ -0,0 +1,598 @@
+(ns tests.basilisp.contrib.nrepl-server-test
+  (:require
+   [basilisp.contrib.bencode :as bc]
+   [basilisp.io :as bio]
+   [basilisp.contrib.nrepl-server :as nr]
+   [basilisp.string :as str :refer [starts-with?]]
+   [basilisp.test :refer [deftest are is testing]])
+  (:import
+   os
+   socket
+   tempfile
+   threading
+   time))
+
+(def ^:dynamic *nrepl-port*
+  "The port the lpy:py:`with-server` is bound to."
+  nil)
+
+(defmacro with-server
+  [opts & body]
+  "Create an nREPL server on a thread with
+  lpy:fn:`basilisp.nrepl-server/server-make` passing in ``opts``, bind
```

Comment:
```suggestion
  :lpy:fn:`basilisp.nrepl-server/server-make` passing in ``opts``, bind
```

Probably won't matter because we won't actually generate docs for this module, but I think rST references need the leading `:`.

I see a couple other instances with the same.

Reply:
Updated all missing `:`s